addons/backups-restic/backups-restic.yaml

```diff
@@ -52,24 +52,14 @@ spec:
               path: /etc/kubernetes/pki
           initContainers:
           - name: snapshoter
-            image: {{ Registry "gcr.io" }}/etcd-development/etcd:v3.5.6
+            image: {{ Registry "gcr.io" }}/etcd-development/etcd:v3.5.11
```

Comment:
We might want to use `k8s.gcr.io` instead in the future.

Reply:
We should, KKP is also still using this image.